ui/narrative/methods/run_velvet/display.yaml

```diff
@@ -1,12 +1,12 @@
 #
 # define display information
 #
-name: Velvet Assembler, Version 1.2.10 
+name: Velvet Assembler - v1.2.10 
 tooltip: |
       Velvet consists of two main parts - Velveth and Velvetg
 screenshots: []
 
-icon: null
+icon: velvet-blue.png
```

Comment:
Where is this icon checked in? I don't see an img directory with the icons, the way it is setup in the trimmomatic directory

Reply:
must have forgotten the "git add" in my batch processing of all these Apps.  Added the image file correctly now.  Thanks for catching!